src/components/header.js

```diff
@@ -135,7 +136,6 @@ function Header({ slug }) {
       >
         <Search
           id="search"
-          className="header__search"
```

Comment:
When there was a min-width, the search bar was too wide to be viewed in the menu navigation panel that opens for tablet/mobile views

Reply:
Sure. If we want to use the search bar in the future, we have to make sure to give it a minimum space to fit ~27 characters (don't remember the exact number).